packages/indexer-common/src/indexer-management/models/indexing-rule.ts

```diff
@@ -13,7 +13,8 @@ export const INDEXING_RULE_GLOBAL = 'global'
 
 export interface IndexingRuleAttributes {
   id: number
-  deployment: string
+  identifier: string | null
```

Comment:
Good point! I also can't think of a case where it would need to be `null`.
When I was making this update I originally was using separate nullable `deployment` and `subgraph` properties and I believe this is a vestige of that strategy. I'll update to be just `string`.